zcash_client_sqlite/src/testing/pool.rs

```diff
@@ -904,7 +914,7 @@ pub(crate) fn spend_succeeds_to_t_addr_zero_change<T: ShieldedPoolTester>() {
     let dfvk = T::test_account_fvk(&st);
 
     // Add funds to the wallet in a single note
-    let value = NonNegativeAmount::const_from_u64(60000);
+    let value = NonNegativeAmount::const_from_u64(70000);
```

Comment:
Why is this amount increased?

Reply:
This change is no longer necessary. I think it was because we were previously calculating the fee incorrectly.